src/app-layer-ssl.c

```diff
@@ -1272,6 +1386,28 @@ static inline int TLSDecodeHSHelloExtensions(SSLState *ssl_state,
                 break;
             }
 
+            case SSL_EXTENSION_SIGNATURE_ALGORITHMS: {
+                /* coverity[tainted_data] */
```

Comment:
What is this coverity thing ?

Reply:
I'd assume it's a code annotation for the static checker Coverity; it's been there. I just kept this in line with the other extension parsers. I agree this may be a bit of cargo cult issue for me; I would be happy to remove it.

Reply:
@victorjulien would you know ?

Reply:
Lets not add it by default, we can add it in if coverity flags this section.

Reply:
Removed.